### PR TITLE
Exclude null-safe plugins from testing on stable

### PR DESCRIPTION
**Pull request by blasten**
Mechanism for disabling the checks on Flutter stable. This needs a PR to the tool.

### CODE REVIEW

File: script/incremental_build.sh [hunk 1]
```diff
@@ -5,13 +5,21 @@ readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
...
   PUB=pub.bat
 else
   PUB=pub
 fi
 
+# Plugins that are excluded from this task.
+ALL_EXCLUDED=("")
+# Exclude nnbd plugins from stable.
+if [[ "$CHANNEL" -eq "stable" ]]; then
+  ALL_EXCLUDED=($EXCLUDED_PLUGINS_FROM_STABLE)
```
Comment (amirh) [inline script/incremental_build.sh:20]: nit: can you print to the log here that we're skipping checks for the following plugins?

File: script/incremental_build.sh [hunk 2]
```diff
@@ -39,18 +47,18 @@ PLUGIN_SHARDING=($PLUGIN_SHARDING)
 
 if [[ "${BRANCH_NAME}" == "master" ]]; then
   echo "Running for all packages"
-  (cd "$REPO_DIR" && $PUB global run flutter_plugin_tools "${ACTIONS[@]}" ${PLUGIN_SHARDING[@]})
+  (cd "$REPO_DIR" && $PUB global run flutter_plugin_tools "${ACTIONS[@]}" --exclude="$ALL_EXCLUDED" ${PLUGIN_SHARDING[@]})
```
Comment (amirh) [inline script/incremental_build.sh:50]: does flutter_plugin_tools support `--exclude`?

**Review by amirh — COMMENTED** [review]
Thanks!!

Generally I'm pushing to do as little as possible in these shell scripts (and ideally move all the logic from here to the flutter_plugin_tools), this is so all of our tests can run on all environments (e.g windows hosts). But as this is a net-improvement and high priority I'm ok with landing it once it works.

File: script/incremental_build.sh (hunk 2 above)
Comment (blasten) [reply script/incremental_build.sh:50]: PR: https://github.com/flutter/plugin_tools/pull/124

File: script/incremental_build.sh (hunk 1 above)
Comment (blasten) [reply script/incremental_build.sh:20]: Done.

**Review by amirh — APPROVED** [review]
LGTM!

**Pull request merged**